### ISSUE TITLE
Build bulk selection UI for transactions

### ISSUE THREAD

**Issue by zachgoll** [maybe-finance/maybe#798]
![Image](https://github.com/maybe-finance/maybe/assets/16676157/492a9059-2e0f-4668-a131-45029f630315)

## Feature Overview

The purpose of this issue is to create UI controls to select 1 or more transactions and perform various actions on those transactions.

## Not in Scope

This issue will NOT cover the actual updates—it is solely _UI focused_.  Placeholders and `cursor-not-allowed` states should be added over the form submissions for now.

## Requirements

_If there is a missing / incorrect requirement, please leave a comment before starting work on this._

- [ ] User can see select checkbox on each transaction
- [ ] User can select 1 or more transactions
- [ ] When selected, a bar at the bottom appears that says how many are selected
  - [ ] User sees an edit icon (placeholder, `cursor-not-allowed`)
  - [ ] User sees a delete icon (placeholder, `cursor-not-allowed`)

## Implementation Suggestions

_Below are some ideas for implementation to get you started.  Use your best judgment here—if there's a better way to do things, go for it!_

## Designs

Below are the designs you should follow while implementing this:
https://www.figma.com/design/lonJmVk3HYkwZoIO7xYP2w/Maybe-App-(Community)?node-id=3626-2572

## Reminders

- Make sure to review our [contributing guidelines](https://github.com/maybe-finance/maybe/blob/main/CONTRIBUTING.md) before starting on an issue
- We do our best to define a clear spec for new features and fixes, but think of them as "suggestions", not "hard requirements".  We welcome ideas and suggestions!
    - If you see missing requirements to this issue, please leave a comment below explaining what is missing and why it is important.
    - If you see a requirement that you think is _incorrect_ or _not optimal_, please leave a comment explaining what you think needs to change below.

**Comment by scubamaggo** [comment]
I have tinkered with this a bit and during the implementation noticed that we have a second transaction list: The one on the dashboard. 
My initial gut feeling when using the checkboxes was that they do not belong on the dashboard view, but should be kept to the Transactions view as seen in the issue description.

What is the general opinion on that?
